packages/internal/src/lib-chainStorage.js

```diff
@@ -73,51 +83,57 @@ harden(assertPathSegment);
  */
 
 /**
- * Create a root storage node for a given backing function and root path.
- *
- * @param {(message: StorageMessage) => any} handleStorageMessage a function for sending a storageMessage object to the storage implementation (cf. golang/cosmos/x/vstorage/vstorage.go)
- * @param {string} rootPath
- * @param {object} [rootOptions]
- * @param {boolean} [rootOptions.sequence] employ a wrapping structure that preserves each value set within a single block, and default child nodes to do the same
+ * @param {import('@agoric/zone').Zone} zone
  */
-export function makeChainStorageRoot(
-  handleStorageMessage,
-  rootPath,
-  rootOptions = {},
-) {
-  assert.typeof(rootPath, 'string');
-
+export const zoneChainStorageNode = zone => {
```

Comment:
Assuming `zone.exoClass` avoids redefinition (at least when `zone` is durable), I think this is a "prepare" pattern (cf. https://github.com/endojs/endo/blob/master/packages/exo/docs/exo-taxonomy.md ). And if it _doesn't_ avoid redefinition, then what happens on upgrade? I think we're missing a README.md for packages/zones.
```suggestion
export const prepareChainStorageNode = zone => {
```

Reply:
> I think we're missing a README.md for packages/zones.

Yep.  Sorry for not marking this PR as draft earlier; it's an experiment to see if `@agoric/zone` is viable before investing too much in it.

Reply:
I still think this `zoneChainStorageNode` function should be renamed to something like `prepareChainStorageNode` in alignment with https://github.com/endojs/endo/blob/master/packages/exo/docs/exo-taxonomy.md .